src/main/resources/static/javascript/mainApp/app.directives.js

```diff
@@ -37,3 +37,30 @@ UHGroupingsApp.directive("tooltipOnTruncate", function () {
         }
     };
 });
+
+UHGroupingsApp.directive("tooltip", function () {
+    return {
+        restrict: "A",
+        scope: {
+            title: "@tooltip"
+        },
+        link(scope, element) {
+            element.attr("data-toggle","tooltip");
+            element.attr("data-original-title", scope.title);
+            element.attr("data-boundary", "window");
+            element.attr("data-delay", "{\"show\": 300}");
+
+            element.on("mouseover", function () {
+                $(element).tooltip("toggle");
+            });
+            element.on("mouseout", function () {
+                $(element).tooltip("dispose");
+            });
+
+        }
+
+        // element.attr("data-placement", attrs["tooltipPosition"]);
+        // this line will make the tooltip nonformatted apparently
```

Comment:
Could you add on to this comment "use Bootstrap's data-placement attribute to change placement of tooltip instead"